crates/holochain_p2p/src/types/wire.rs

```diff
@@ -66,42 +79,63 @@ pub enum WireMessage {
         flag: bool,
         op: DhtOp,
     },
+    */
+}
+
+fn next_msg_id() -> u64 {
+    static M: std::sync::atomic::AtomicU64 = std::sync::atomic::AtomicU64::new(1);
+    M.fetch_add(1, std::sync::atomic::Ordering::Relaxed)
```

Comment:
Relaxed ordering okay? Zome calls made in parallel from the UI or multiple clients operating at the same time?

Reply:
Yep, no problem with Relaxed. It doesn't matter if two parallel threads get numbers out of sequence to when they requested it, because the only invariant is that the message ids are unique.

Reply:
Right, yes that makes sense!